.github/workflows/build.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
         os: [macos-latest, ubuntu-latest, windows-latest]
 
         golang_version: [1.19.3]
-        java_version: ['15', '16', '17']
+        java_version: ['15', '16', '17', '18', '19']
```

Comment:
Since I removed the test annotations, I'd also like to show that the tool works with even more recent Java versions than 17